### ISSUE TITLE
FX function normalization broken for some cases due to FX operator schema change

### ISSUE THREAD

**Issue by jfix71** [pytorch/pytorch#76462]
### 🐛 Describe the bug

Certain ops are no longer normalized correctly due to the changes in https://github.com/pytorch/pytorch/pull/75896

I created https://github.com/pytorch/pytorch/pull/76457 to consider backing out https://github.com/pytorch/pytorch/pull/75896. The PR includes the below test added to `test/test_fx_experimental.py` which exhibits the issue:

```python
    def test_normalize_quantized_eb(self):
        target = torch.ops.quantized.embedding_bag_byte_rowwise_offsets
        args = (
            torch.empty((2, 3), dtype=torch.uint8),
            torch.empty((2,), dtype=torch.int64),
            torch.empty((2,), dtype=torch.int64),
        )
        norm_args_and_kwargs = normalize_function(
            target, args, normalize_to_only_use_kwargs=True
        )
        self.assertTrue(norm_args_and_kwargs is not None)
        self.assertEqual(
            set(norm_args_and_kwargs.kwargs.keys()),
            {
                "weight",
                "indices",
                "offsets",
                "scale_grad_by_freq",
                "mode",
                "pruned_weights",
                "per_sample_weights",
                "compressed_indices_mapping",
                "include_last_offset",
            },
        )
        self.assertEqual(norm_args_and_kwargs.args, tuple())
```

### Versions

Collecting environment information...
PyTorch version: N/A
Is debug build: N/A
CUDA used to build PyTorch: N/A
ROCM used to build PyTorch: N/A

OS: CentOS Stream 8 (x86_64)
GCC version: (GCC) 8.5.0 20210514 (Red Hat 8.5.0-7)
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.30

Python version: 3.8.6 (default, Jun  4 2021, 05:16:01)  [GCC 9.x 20210412 (Facebook) 10.x] (64-bit runtime)
Python platform: Linux-5.12.0-0_fbk5_hardened_4818_g9939bf8c1268-x86_64-with-glibc2.2.5
Is CUDA available: N/A
CUDA runtime version: Could not collect
GPU models and configuration: Could not collect
Nvidia driver version: Could not collect
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: N/A

Versions of relevant libraries:
[pip3] Could not collect
[conda] Could not collect

**Comment by jfix71** [comment]
Hi @peterbell10, do you have any quick thoughts on the right fix here? This is causing some issues for some internal use cases and we'd love to fix it quickly if possible, or else may try to back out the original PR.

**Comment by peterbell10** [comment]
Fixed by #76469.
